### DISCUSSION TITLE
testing is failed with "Error: `url` parameter is mandatory if no client is provided"

### DISCUSSION THREAD

**Discussion by crapthings** [fastify/fastify#4115] [Q&A] [answered]
i'm following the testing docs, but get errors

<img width="1062" alt="image" src="https://user-images.githubusercontent.com/1147704/177308981-5e9f17e3-6bda-4a2a-b652-0953ace47507.png">

```js
const { test } = require('tap')

const Fastify = require('../src')

test(`requests the '/api/status' route`, async (t) => {
  const fastify = Fastify()

  t.teardown(() => fastify.close())

  const resp = await fastify.inject({
    method: 'GET',
    url: '/api/status',
  })

  // console.log('status code: ', resp.statusCode)
  // console.log('body: ', resp.body)

  // t.equal(resp.statusCode, 200, 'returns a status code of 200')
})
```

**Comment by mcollina** [comment]
Thanks for reporting!

Can you provide steps to reproduce? We often need a [reproducible example](https://stackoverflow.com/help/minimal-reproducible-example), e.g. some code that allows someone else to recreate your problem by just copying and pasting it. If it involves more than a couple of different file, create a new repository on GitHub and add a link to that.

**Comment by crapthings** [comment, marked as answer]
that's just i forgot to give mongourl @fastify/mongodb

everything works after add "require('dotenv').config()"

I thought it was url from inject().get()
